core/src/main/java/io/questdb/network/IOContextFactoryImpl.java

```diff
@@ -48,7 +47,6 @@ public void close() {
         closed = true;
     }
 
-    @Override
```

Comment:
same, annotation should not have been removed

Reply:
restored back